boards/arm/cy8cproto_062_4343w/doc/index.rst

```diff
@@ -0,0 +1,205 @@
+| Interface | Controller | Driver/Component      |
++===========+============+=======================+
+| NVIC      | on-chip    | nested vectored       |
+|           |            | interrupt controller  |
++-----------+------------+-----------------------+
+| SYSTICK   | on-chip    | system clock          |
++-----------+------------+-----------------------+
+| GPIO      | on-chip    | GPIO                  |
++-----------+------------+-----------------------+
+| UART      | on-chip    | serial port-polling;  |
+|           |            | serial port-interrupt |
++-----------+------------+-----------------------+
+
+
+The default configuration can be found in the Kconfig
+
+:zephyr_file:`boards/arm/cy8cproto_062_4343w/cy8cproto_062_4343w_defconfig`
+
+
+System Clock
+============
+
+The PSoC 62 MCU SoC is configured to use the internal IMO+FLL as a source for
+the system clock. CM0+ works at 50MHz, CM4 - at 100MHz. Other sources for the
+system clock are provided in the SOC, depending on your system requirements.
+
+
+Install Infineon Connectivity Zephyr module
+*******************************************
+This modules includes assets which has any binary files (e.g WIFI/Bluetooth chip firmware,
+CM0p prebuilt images, etc).
+
+The following steps shows how to enable Infineon Connectivity Zephyr module in project:
+
+1. Get the Zephyr source code
+
+.. code-block:: console
+
+    west init
+
+2. Add an infineon_connectivity.yaml manifest file in to zephyr/submanifests folder. There is an example yaml file in this directory.
+Use the snippet of infineon_connectivity.yaml below:
+
+.. code-block:: yaml
+
+   manifest:
+     projects:
+       - name: infineon_zephyr_connectivity
+          url: https://github.com/Infineon/infineon_zephyr_connectivity.git
+          submodules: true
+          revision: develop
+          path: modules/hal/infineon_connectivity
+          groups:
+            - hal
+
+
+3. Update Zephyr modules
+
+.. code-block:: console
+
+   west update
+
+3. Update Zephyr modules and export the environment
+
+.. code-block:: console
+
+   west update
+   west zephyr-export
+
+
+Build blinking led sample
+*************************
+Here is an example for the :ref:`blinky-sample` application.
+
+.. code-block:: console
+
+   cd zephyr
+   west build -p auto -b cy8cproto_062_4343w samples/basic/blink
+
+OpenOCD Installation
+====================
+
+You must download `Infineon OpenOCD`_ from Github to flash and debug.
+Extract the files and note the path.
+
+Programming and Debugging
+*************************
+
+The CY8CPROTO-062-4343W includes an onboard programmer/debugger (KitProg2) with
+mass storage programming to provide debugging, flash programming, and serial
+communication over USB. Flash and debug commands must be pointed to the Cypress
+OpenOCD you downloaded above.
+
+On Windows:
+
+.. code-block:: console
+
+   west flash --openocd /path/to/your/openocd/bin/openocd.exe
```

Comment:
I would suggest you instead use something like `path/to/infineon/openocd/bin/openocd.exe`

Reply:
done